modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7536,6 +7536,8 @@ feature.flag.LPD-31212.description=Users can create rich text content that inclu
 feature.flag.LPD-31212.title=Disable XSS Validations in Rich Text Editor
 feature.flag.LPD-31789.description=Allow administrators to configure Google Vertex AI as text embedding provider for semantic search.
 feature.flag.LPD-31789.title=Google Vertex AI Text Embedding Provider
+feature.flag.LPD-34594.description=Users can create object definition hierarchies by defining a root object definition. Child object definitions are bound to the root and inherit permissions and widget configuration from the root object definition. The API endpoints of child object definitions are merged with the root's endpoints.
+feature.flag.LPD-34594.title=Root Objects
```

Comment:
@devpedrotavares don't we need a `buildLang` commit?

Reply:
Yes we do. I forgot to add it. Thank you for pointing it out!